docs/extensions/tutorials/_posts/1970-01-01-GettingStarted.md

```diff
@@ -21,47 +21,36 @@ Shoutem prepared and [open sourced](https://github.com/shoutem/extensions) a lot
+- [Node.js and npm](https://nodejs.org/en/download/) (installing `Node.js` also installs `npm`)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- react-native-cli - `$ npm install -g react-native-cli`
+- shoutem-cli - `$ npm install -g @shoutem/cli`
 
-<p class="image">
-<img src='{{ site.url }}/img/tutorials/getting-started/shoutem-preview-app.png'/>
-</p>
+> #### Note
+> If the `npm install` commands fail because of _permission_ issues, you need to run them with `sudo`: `sudo npm install -g <package-name>`.
 
-## Managing Your App
+If you haven't already, go to the [Builder]({{ site.shoutem.builderURL }}) and create an account. Once signed in, create a new `Blank app`. For this quick tutorial, we'll make a Restaurants and Food app, lets call it `Restaurants`. To rename your app, click on the `Blank app` text, delete it and type in `Restaurants`.
 
-You can manage your app inside the Builder. The app structure can be seen under **Screens**. It starts with **Main navigation** which has multiple screens nested inside. Those are the screens from extensions you have installed which you can add by clicking on `+`. For the purpose of the tutorial, delete the **About** screen by selecting it, clicking on the three dots (`...`) on the top right corner and selecting **Delete**.
+Lets say we want to add an RSS feed screen to our app. To do that, click on the + button next to Screens and select the News RSS extension. Since the app is about Restaurants, to stick to the Food theme, lets use a food related News RSS feed:
+http://foodengineeringmag.com/rss/topic/2639-top-100-food-beverage-companies
+
+After you add this screen, it should look something like this:
 
 <p class="image">
-<img src='{{ site.url }}/img/tutorials/getting-started/delete-starting-screen.png'/>
+<img src='{{ site.url }}/img/tutorials/getting-started/01-builder-before-clone.png'/>
 </p>
 
-> #### Note
-> All Shoutem extensions are pre-installed to your app on the Builder so you can use their screens right away, but any custom extensions you make will have to be installed using `shoutem install`, before you can use them through the Builder.
-
-Let's create a new extension with a `Hello World` screen and add it to the app.
-
 ## Creating a New Extension
 
-Start by installing the [Shoutem CLI]({{ site.shoutem.cli }}) - tool that makes developing extensions a breeze.
-
-```ShellSession
-$ npm install -g @shoutem/cli
-```
-
-> #### Note
-> If previous command fails because of _permission_ issues, you need to run it with `sudo` permission: `sudo npm install -g @shoutem/cli`.
-
-Register your developer name. Use the `shoutem login` command with your Shoutem credentials ("{{ site.example.devName }}" is used as a developer name in this example).
+Start by using the `shoutem login` command with your Shoutem credentials ("{{ site.example.devName }}" is used as a developer name in this example).
```

Comment:
Added this, not sure why the change wasn't covered. Probably because it was added in a different line.